packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -0,0 +1,437 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ResizeObserver from 'resize-observer-polyfill';
+import List from 'terra-list';
+import SelectableUtils from 'terra-list/lib/SelectableUtils';
+import InfiniteUtils from './_InfiniteUtils';
+import styles from './InfiniteList.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child terra-list ListItems to be placed within the infinite list.
```

Comment:
I think having terra-list in the document causes confusion as to what to inject into the children prop. Perhaps we can update the doc to tell developers they should be generating their children list with InfiniteList.Item?